RecoCTPPS/TotemRPLocal/plugins/CTPPSLocalTrackLiteProducer.cc

```diff
@@ -26,20 +27,21 @@
  **/
 class CTPPSLocalTrackLiteProducer : public edm::stream::EDProducer<>
 {
-  public:
-    explicit CTPPSLocalTrackLiteProducer( const edm::ParameterSet& );
-    virtual ~CTPPSLocalTrackLiteProducer() {}
+public:
```

Comment:
Hi all! If another iteration is required (hope not, though :) ), would it be possible to revert back to the previous indentation scheme (as a matter of consistency)?